### PR TITLE
Add missing documentation for 64-bit signed integers used as doc attributes

### PR DESCRIPTION
**Pull request by lfarcy**
Also removes useless trailing white-spaces in documentation files.

### CODE REVIEW

File: doc/sphinx.txt [hunk 1]
```diff
@@ -4108,10 +4110,10 @@ properly.
 
 Second, we additionally need to know the language-specific string sorting
 rules. For instance, when sorting according to US rules in en_US locale,
-the accented character '�' (small letter i with diaeresis) should be placed
+the accented character '�' (small letter i with diaeresis) should be placed
```
Comment (Koc) [inline doc/sphinx.txt]: looks wrong

Comment (lfarcy) [inline doc/sphinx.txt]: Konstantin,

I will look at what went wrong with the encoding on my side. In the meantime, can you please tell me what encoding those documentation files use ?

Thx.

Comment (Koc) [inline doc/sphinx.txt]: utf8 or cp1251 (windows-1251) maybe

**Comment by lfarcy** [comment]
I'm going to redo the modifications from scratch and make sure I use the right encoding. Closing this PR...

**Pull request closed without merging**